subworkflows/local/mtx_conversion.nf

```diff
@@ -14,14 +14,6 @@ workflow MTX_CONVERSION {
     main:
         ch_versions = Channel.empty()
 
-        // Cellranger module output contains too many files which cause path collisions, we filter to the ones we need.
-        if (params.aligner in [ 'cellranger', 'cellrangerarc' ]) {
```

Comment:
Hi @apeltzer  and @heylf ,
This was removed from here, because it was to be passed to the sub-workflows, where at the end of `cellranger` for example, the parsing occurs so that we know what is filtered and what is raw.

Is something like this required for `cellrangerarc` as well? Or the way it is right now is working for `cellrangerarc`? This is my only fear that `cellrangerarc` may fail now.

Reply:
@heylf is the right person to ask here, have not tested this myself unfortunately.

Reply:
Hi @apeltzer ,
Since there was no reply. Should it be merged and if is not working then is fixed in a new issue?

Reply:
Well, I guess we'd need https://github.com/nf-core/scrnaseq/issues/290. I suspect that you are right and something is broken for cellranger-arc now.

Reply:
Hi @grst and @apeltzer ,
I created a PR #321 to include the snippet back for `cellranger-arc` so we keep as it was for release.

FYI: @heylf